frontend/ui-pablo/app/components/Organisms/YourBondTable.tsx

```diff
@@ -79,39 +78,47 @@ export const YourBondTable: React.FC = () => {
             </TableRow>
           </TableHead>
           <TableBody>
-            {pools.map((pool, index) => (
+            {activeBonds.map((bond, index) => (
               <TableRow
-                onClick={handleRowClick}
+                onClick={() => handleRowClick(bond.offerId)}
                 key={index}
-                sx={{cursor: "pointer"}}
+                sx={{ cursor: "pointer" }}
               >
                 <TableCell align="left">
-                  {pool.token2 ? (
+                  {"base" in bond.asset ? (
```

Comment:
"base" in `bond.asset` only checks for the existence of that key. I suggest we change this to actually check the presence of a valid object.

```ts
bond.asset?.base ? (<PairAsset ... />) : ...
```

Reply:
yup but can be changed in future as YourBonds havent been integrated yet